Source/Core/ClippingPlaneCollection.js

```diff
@@ -574,27 +570,62 @@ define([
      */
     ClippingPlaneCollection.TEXTURE_WIDTH = CesiumMath.nextPowerOfTwo(Math.ceil(Math.sqrt(ClippingPlaneCollection.MAX_CLIPPING_PLANES * 2)));
 
+    /**
+     * Returns true if this object was destroyed; otherwise, false.
+     * <br /><br />
+     * If this object was destroyed, it should not be used; calling any function other than
+     * <code>isDestroyed</code> will result in a {@link DeveloperError} exception.
+     *
+     * @returns {Boolean} <code>true</code> if this object was destroyed; otherwise, <code>false</code>.
+     *
+     * @see ClippingPlaneCollection#destroy
+     */
     ClippingPlaneCollection.prototype.isDestroyed = function() {
         return false;
     };
 
     /**
-     * Destroys this ClippingPlaneCollection if it either has no owner or a reference to the owner is passed in.
-     * We only expect an owner to be defined if this collection is shared by one Cesium3DTileset
-     * and a series of Cesium3DTiles with Model content.
+     * Destroys this ClippingPlaneCollectionand its WebGL resources.
      * @private
+     */
+    ClippingPlaneCollection.prototype.destroy = function() {
+        this._clippingPlanesTexture = this._clippingPlanesTexture && this._clippingPlanesTexture.destroy();
+        return destroyObject(this);
+    };
+
+    /**
+     * Sets the owner for the input ClippingPlaneCollection if there wasn't another owner.
+     * Destroys the owner's previous ClippingPlaneCollection if setting is successful.
      *
-     * @param {Object} owner An object to check against this ClippingPlaneCollection's owner object
+     * @param {ClippingPlaneCollection} [clippingPlaneCollection] A ClippingPlaneCollection (or undefined) being attached to an object
+     * @param {Object} newOwner An Object that should receive the new ClippingPlaneCollection
+     * @param {String} key The Key for the Object to reference the ClippingPlaneCollection
+     * @private
      */
-    ClippingPlaneCollection.prototype.checkDestroy = function(owner) {
-        if (defined(this.owner) && owner !== this.owner) {
-            return this;
+    ClippingPlaneCollection.setOwnership = function(clippingPlaneCollection, newOwner, key) {
```

Comment:
Place `setOwnership` and `isSupported` above the destroy functions, which are usually last.